test/vtl.test.ts

```diff
@@ -359,6 +359,34 @@ test("$util.time.nowISO8601", () =>
     })
   ));
 
+test("$util.log.info(message)", () =>
+  appsyncTestCase(
+    reflect(() => {
+      return $util.log.info("hello world");
+    })
+  ));
```

Comment:
Does this create snapshot tests? I don't see them in the PR

Reply:
I can't run the tests locally, for some reason. It fails due to AWS account number not being valid. Is there another command to record the snapshots?

Reply:
Oh have you set up local stack? Our tests spin that up locally and deploy stacks to it. https://localstack.cloud/

We need to set up a contributing guide. Thanks for the reminder.

Reply:
Tracking https://github.com/functionless/functionless/issues/234